### DISCUSSION TITLE
What exactly does max_traces_per_user mean?

### DISCUSSION THREAD

**Discussion by manojm321** [grafana/tempo#2880] [Q&A] [answered]
Hello, 

```yaml
    # Maximum number of active traces per user, per ingester. A value of 0
    # disables the check.
    # Results in errors like
    #    LIVE_TRACES_EXCEEDED: max live traces per tenant exceeded:
    #    per-user traces limit (local: 10000 global: 0 actual local: 1) exceeded
    # This override limit is used by the ingester.
    [max_traces_per_user: <int> | default = 10000]
```
does `user` map to `service.name`?

**Comment by mapno** [comment, marked as answer]
Hi! `user` and `tenant` are used interchangeably in Tempo. A tenant is defined by a header during ingestion, refer to the [docs](https://grafana.com/docs/tempo/latest/operations/multitenancy/) to enable multi-tenancy.

**Reply by manojm321** [reply]
Thanks, can I set a limit per servce.name?

**Reply by mapno** [reply]
Unfortunately, no. Limits are configured per tenant.
